spec/acceptance/letsencrypt_plugin_dns_rfc2136_spec.rb

```diff
@@ -3,8 +3,7 @@
 describe 'letsencrypt::plugin::dns_rfc2136' do
   supported = case fact('os.family')
```

Comment:
You could also write the whole definition now as:
```ruby
supported = ['Debian', 'RedHat'].include?(fact('os.family'))
```

You can even natively skip things:
```ruby
 describe 'letsencrypt::plugin::dns_rfc2136', if: ['Debian', 'RedHat'].include?(fact('os.family')) do
```

Then you can drop `if supported` on line 28` as well.

Reply:
It didn't come to my mind, that a named parameter would be called `if`. The ruby irc helped me understand. :)

I tried to implement the if filter.

For me the question came up, is it not possible to support this two certbot plugins on the BSD systems supported by this module? This would improve the tests even more.

Reply:
Maybe? I don't know BSD. If there's a package then yes. We can also argue that we don't run acceptance tests on BSD at all now so the whole `if:` could be dropped. If we do add BSD, then we can deal with it then?